src/moderatorFrontendMain/kotlin/views/login/MailLogin.kt

```diff
@@ -34,16 +36,28 @@ interface MailLoginState : RState {
   var errorMessage: String?
 }
 
-class MailLogin : LoginDetailComponent<MailLoginProps, MailLoginState>() {
-  override fun MailLoginState.init() {
+class MailLogin(props: MailLoginProps) : LoginDetailComponent<MailLoginProps, MailLoginState>(props) {
+  override fun MailLoginState.init(props: MailLoginProps) {
     email = ""
     password = ""
     networkRequestInProgress = false
     errorMessage = null
+
+    // Autologin if in demo mode
```

Comment:
I would not do an autologin but just an auto-prefilling in demo-mode. I guess it could be from an UX perspective confusing if after 1.5 seconds the login-view automatically disappears.